src/Controls/src/Core/ActivityIndicator/ActivityIndicator.cs

```diff
@@ -40,5 +42,10 @@ public IPlatformElementConfiguration<T, ActivityIndicator> On<T>() where T : ICo
 		{
 			return _platformConfigurationRegistry.Value.On<T>();
 		}
+
+		private protected override string GetDebuggerDisplay()
+		{
+			return $"IsRunning = {IsRunning}, " + base.GetDebuggerDisplay();
```

Comment:
Interpolating along with string concatenation ?

Reply:
would it cause any issue?

Reply:
String concatenation should never be use in modern c#